sunpy/map/sources/tests/test_swap_source.py

```diff
@@ -7,20 +7,26 @@
 
 from sunpy.data.test import get_dummy_map_from_header, get_test_filepath
 from sunpy.map.sources.proba2 import SWAPMap
-
-header_files = [
```

Comment:
What are the differences between these files?

Reply:
Time. It's just a sequence of files.